lib/plistLib/main.js

```diff
@@ -250,8 +252,8 @@ define([
             dict: function(doc, dict) {
                 var fragment = doc.createDocumentFragment(),
                     key, value, keyElement;
-                for(key in dict) { 
-                    if(!dict.hasOwnProperty(key) || dict[key] === undefined || dict[key] === null)
+                for(key in dict) {
+                    if(!Object.prototype.hasOwnProperty.call(dict, key) || dict[key] === undefined || dict[key] === null)
```

Comment:
I described above my reasoning for creating objects as key/value storage like `Object.create(null)` In this case I allowed such Objects to be passed as arguments to plist serialization. This implementation is probably not very smart and just a reaction to a bug I had (fixed it though).

Better would be to check whether `Object.prototype` is the beginning of the prototype chain and then use the old code or if `null` is the beginning of the prototype chain and then just use `for key in`. This would work with what I wrote above, how I expect a "dict" to behave, i.e. that its prototypes are also part of the data.